docs/api/core/Raycaster.html

```diff
@@ -144,12 +144,12 @@ <h3>[method:null setFromCamera]( [param:Vector2 coords], [param:Camera camera] )
 		Updates the ray with a new origin and direction.
 		</div>
 
-
-		<h3>[method:Array intersectObject]( [param:Object3D object], [param:Boolean recursive] )</h3>
+		<h3>[method:Array intersectObject]( [page:Object3D object], [param:Boolean recursive], [param:Array optionalTarget] )</h3>
 		<div>
 		<p>
 		[page:Object3D object] — The object to check for intersection with the ray.<br />
-		[page:Boolean recursive] — If true, it also checks all descendants. Otherwise it only checks intersecton with the object. Default is false.
+		[page:Boolean recursive] — If true, it also checks all descendants. Otherwise it only checks intersecton with the object. Default is false.<br />
+		[page:Array optionalTarget] — (optional) target to set the result. Otherwise a new [page:Array] is instantiated. Application is responsible for clearing this array on each call (e.g., array.length = 0;).
```

Comment:
Instead of

    Application is responsible for clearing this array on each call (e.g., array.length = 0;).

It may be more clear like so:

    If set, you must clear this array prior to each call (e.g., array.length = 0;).